README.md

```diff
@@ -44,24 +44,32 @@ esy test
 
 ### Running a sidechain
 
-For convenient local development, we have a included a script `sandbox.sh` that
-automates the setup of a local Tezos network using [Flextesa](https://tezos.gitlab.io/flextesa/)
-running in Docker. Additionally, the script sets up a identities for a local Deku cluster.
-This script is the easiest way to get started with Deku; however, it uses unsafe
-configuration options to lower the required Tezos confirmations to 1. This setting greatly
-speeds up local development, **but is not at all safe for production**!
+For convenient local development, we have a included two components:
+- A `docker-compose.yml` file that setups a local Tezos network
+  using [Flextesa](https://tezos.gitlab.io/flextesa/) running in Docker.
+  Additionally, the network sets up a [Better Call Dev](https://github.com/baking-bad/bcdhub) instance
+  for introspection of the deployed contract and its operations. The BCD interace is available in
+  your browser at http://localhost:8000.
+- A script `./sandbox.sh` that sets up a identities for a local Deku cluster.
+  This script is the easiest way to get started with Deku; however, it uses unsafe
+  configuration options to lower the required Tezos confirmations to 1. This setting greatly
+  speeds up local development, **but is not at all safe for production**!
 
 #### Requirements
 
-The sandbox requires only Bash and [Docker](https://docs.docker.com/get-docker/) to be installed.
-
+The sandbox requires Bash, [Docker](https://docs.docker.com/get-docker/), and docker-compose to be installed,
```

Comment:
What about a `nix-shell` to provide all these binaries?

Reply:
Hello fellow Nix friend! :raised_hands: 

I'm not sure that, e.g. `nix-shell -p docker` is enough to get a working docker setup, since it requires the docker daemon to be configured and running, etc. As for bash, I think this is sufficiently ubiquitous as to not need a Nix. In fact, I think people might be annoyed if they had to install Nix and only used it to install bash :sweat_smile: 

We could create a pure Nix setup that removes Docker entirely, but then we lose the advantage of better-call.dev maintaining and supporting this docker-compose setup for us.

In short, I'm definitely in favor of building Deku with Nix (#438 is great, and I'm going to test it shortly), but I think Docker is a reasonable system requirement for running a Deku cluster in development mode.

Reply:
Thanks for this detailed explanation dear Nix friend 😉
I was not going so deep, but just thinking about providing the necessary binaries to run our project. I am tired of pre-requisites list with tons of tools to install.
For now, I think [`esy`](https://search.nixos.org/packages?channel=unstable&show=nodePackages.esy&from=0&size=50&sort=relevance&type=packages&query=esy), [`docker`](https://search.nixos.org/packages?channel=unstable&show=docker&from=0&size=50&sort=relevance&type=packages&query=docker), [`docker-compose`](https://search.nixos.org/packages?channel=unstable&show=docker-compose&from=0&size=50&sort=relevance&type=packages&query=docker-compose) and maybe [`opam`](https://search.nixos.org/packages?channel=unstable&show=opam&from=0&size=50&sort=relevance&type=packages&query=opam).
Providing them within a nix-shell, would allow any new people to simply do:
```shell script
$ cd deku
$ nix-shell
$ docker-compose up -d
```
without saying 
> if you want to contribute, please install .......

You can have a look at this [shell.nix](https://github.com/gsebil08/merkle-tree/blob/master/shell.nix) I provided on my merkle-tree test fr marigold 😄

Reply:
@gsebil08 the only requirement is `esy`, `opam` is not used anywhere, for a long time I didn't have installed nor `docker` or `docker-compose either`.